dbt_project.yml

```diff
@@ -1,6 +1,8 @@
 name: 'dbt_utils'
 version: '0.1.0'
 
+require-dbt-version: "0.13.0"
```

Comment:
We can set this to a version range, such that dbt-utils will work with future versions of dbt!

```suggestion
require-dbt-version: ">=0.13.0"
```

Reply:
Makes sense. Done!